gulpfile.js

```diff
@@ -201,7 +201,7 @@ var syncDepenedents = [
 
 gulp.task('sync', syncDepenedents, function() {
   sync.init(null, {
-    proxy: 'http://localhost:3000',
+    proxy: 'http://0.0.0.0:3000',
```

Comment:
I haven't used docker, but yeah on C9, you do have to setup 0.0.0.0 as your IP. Might I suggest we make this an ENV var and add a `require('dotenv').load();` to the top.

Of course with fallbacks to localhost ?